### ISSUE TITLE
Cartesian Chart does not update after pan gesture

### ISSUE THREAD

**Issue by friebetill** [syncfusion/flutter-examples#83]
If one adjusts the zoom factor after panning the graph, the zoom factor is not included in the calculation again.

I made a [video of the error](https://imgur.com/a/4YxLeeE). The video shows two runs, one without panning and one with panning. If there is no pan, it works fine.

It is implemented so that the graph always starts in zoomed mode. In portrait mode the zoom should be 0.6 and in landscape mode 0.2, but if one pans the graph first, the zoom factor is ignored when the screen is rotated. 

Thanks for the great library!

**Comment by dharanidharandharmasivam** [comment]
Hi @friebetill ,

Greetings from Syncfusion. We have analyzed your scenario with the provided video. We would like to let you know that, once you have rendered the chart with zoom factor as 0.6 in portrait mode, then on user interactions like panning, zooming, etc we will maintain the state. So that on orientation change the state will be maintained and this is our default behavior of chart widget. Therefore, on orientation change after panning, the state got maintained in your case. 

We can achieve your requirement as a workaround using the global key, we have cleared the zoomed axis states so that the states will not be maintained. Find the code snippet below

> 
    SfCartesianChart chart;
    GlobalKey chartKey = GlobalKey<State>();
    chart =  SfCartesianChart(
      key:chartKey,
      //Other Configurations
    );
    OrientationBuilder(
            builder: (context, orientation) {
              if(MediaQuery.of(context).orientation == Orientation.landscape){
               dynamic chartState = chartKey.currentState;
               chartState.zoomedAxisStates.clear();
              }
              return Center(
                child: Container(
                  child:chart,
                ),
              );
            })

The sample for reference can be found below.
https://www.syncfusion.com/downloads/support/directtrac/general/ze/chart_sample781380945 

`Note: Since we are clearing the zoomed axis states, the state will not be maintained. For example, if you have panned in the portrait mode, then on orientation change to the landscape and again coming back to portrait mode, the panned state will not be maintained now. So, we request you to either use our default behavior or the above workaround.`

Thanks,
Dharani.
